buildkite/bazelci.py

```diff
@@ -980,7 +980,7 @@ def execute_bazel_test(bazel_binary, platform, flags, targets, bep_file, monitor
         elif not monitor_flaky_tests:
             aggregated_flags += remote_caching_flags(platform)
     aggregated_flags += flags
-    aggregated_flags += incompatible_flags
+    aggregated_flags += (incompatible_flags or [])
```

Comment:
Nit: same as above. I'll send a PR to eliminate the duplicate code here.

Reply:
Thanks!